### DISCUSSION TITLE
LE BAG : ON PEUT S'EN PASSER

### DISCUSSION THREAD

**Discussion by Tournesol01** [A-S-T-U-C-E/STudio4Education#50] [Ideas]
Autre suggestion 

LE BAG : ON PEUT S'EN PASSER

Le menu contextuel propose «Disable Block».

D'autre part, il suffit de parquer le ou les blocs désactivés dans une zone un peu éloignée du sketch pour disposer le cas échéant d'une «banque de blocs locale».
Ils sont grisés et donc très repérables.
À la rigueur, s'ils étaient à peine moins grisés, leur identification en serait facilitée.
⇒ moins compliqué que l'utilisation du bag.

**Comment by SebCanet** [comment]
Développé pour Scratch et APpInventor, bien plus efficace que d'avoir une zone de travail encombrée, dont on ne sait plus si ça doit faire partie du programme ou bien si c'est en stock. Mais je réfléchis à une autre ergonomie pour que la bagpack soit en fait une catégorie dans la toolbox.
Il faut penser ergonomie pour les débutants, du visuel avant tout.
